File: Program/Tier1/Shared/DTOs/HousingIdDTO.cs
namespace Shared.DTOs;

public class HousingIdDTO
{
    public long HousingId { get; }
    public string HostEmail { get; }
    public string ErrorMessage { get; }

    public HousingIdDTO(long housingId, string hostEmail, string errorMessage)
    {
        HousingId = housingId;
        HostEmail = hostEmail;
        ErrorMessage = errorMessage;
    }
}

File: Program/Tier2/src/main/java/via/sep3/group11/tier2/shared/DTOs/HousingIdDTO.java
package via.sep3.group11.tier2.shared.DTOs;

public class HousingIdDTO {
    private Long housingId;
    private String hostEmail;
    private String errorMessage;

    public HousingIdDTO(Long housingId, String hostEmail, String errorMessage) {
        this.housingId = housingId;
        this.hostEmail = hostEmail;
        this.errorMessage = errorMessage;
    }

    public Long getHousingId() {
        return housingId;
    }

    public String getHostEmail() {
        return hostEmail;
    }

    public String getErrorMessage() {
        return errorMessage;
    }
}
